modules/y_lb_main_menu_cta_block/y_lb_main_menu_cta_block.install

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @file
+ * Install, update and uninstall functions for the Website Services Main Menu CTA block module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function y_lb_main_menu_cta_block_install(): void {
+  \Drupal::messenger()->addStatus(t('Module Website Services Main Menu CTA block has been installed.'));
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function y_lb_main_menu_cta_block_uninstall(): void {
+  \Drupal::messenger()->addStatus(t('Module Website Services Main Menu CTA block has been uninstalled.'));
```

Comment:
No need for this 
![image](https://github.com/user-attachments/assets/6306b9ec-387d-4fee-8dc6-9e31abb180a5)